include/zproc.h

```diff
@@ -95,7 +95,7 @@ CZMQ_EXPORT void *
     zproc_stderr (zproc_t *self);
 
 //  *** Draft method, for development use, may change without warning ***
-//  Starts the process.
+//  Starts the process, return just before execve/CreateProcess
```

Comment:
note that if you don't add this to the api/zproc.api file it will be overwritten